OpenMarket/OpenMarket/Utils/NumberFormatter.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  NumberFormatter.swift
+//  OpenMarket
+//
+//  Created by 박병호 on 2022/01/13.
+//
+
+import Foundation
+
+struct PresentStyle {
+  static var numberFormatter: NumberFormatter {
+    let formatter = NumberFormatter()
+    formatter.numberStyle = .decimal
+    
+    return formatter
+  }
```

Comment:
지금 적용한 방식은 이 프로퍼티를 호출할 때마다 새로운 포매터를 생성하여 돌려주는 방식으로 작동할거에요. 의도한대로 인스턴스를 한 번만 만들어 사용하고자 한다면 아래 형식으로 사용할 수 있어요.

```suggestion
  static let numberFormatter: NumberFormatter = {
    let formatter = NumberFormatter()
    formatter.numberStyle = .decimal
    return formatter
  }()
```